spot-client/src/spot-remote/ui/components/dial-pad/StatelessDialPad.js

```diff
@@ -126,7 +127,7 @@ export default class StatelessDialPad extends React.Component {
                         { this._renderDialButton('9', 'WXYZ') }
                     </div>
                     <div className = 'row'>
-                        { this._renderDialButton('＊') }
+                        { this._renderDialButton('*') }
```

Comment:
Does this look okay? I tried matching the designs and the asterisk didn't look good, which is what led to the bug. I was thinking we might need to add support for showing one value but having another value that's actually used.

Reply:
Okay I can try doing that

Reply:
Yeah that character looks better somehow